### PR TITLE
Assignment 5 Nicholas Burlakov Dillon Pikulik

### PR DESCRIPTION
**Pull request by NicholasBurlakov**
https://ambitious-grass-04916f41e.3.azurestaticapps.net/
Nicholas Burlakov and Dillon Pikulik
Several things do not work. Adding duplicate words is still allowed, MasterOfTheUniverse is not implemented nor is apriori. Other than that we have a solid start!

### CODE REVIEW

File: wordle-web/src/views/LoginView.vue
Comment (schemcharger) [inline wordle-web/src/views/LoginView.vue:1]: logging in with incorrect credentials routes me to the main page.

**Review by schemcharger — COMMENTED** [review]
Create an editor for the available words ✅
Create a page named "WordEditor" that allows words to be edited ✅
Support deleting a word ✅
Support adding a new word ✅
Editing words is not supported ✅
Duplicate words should not be allowed ❌
Support changing the common word boolean flag ❌✅ - I can't tell if it is working
Words should be sorted alphabetically ❌
Provide a text box for searching words AS YOU TYPE which supports searching for words starting with letters specified ❌
Provide a menu item for navigating to the WordEditor page ✅
Anyone can look at the word list ✅
Word list is paginated (10-100 per page) ✅
Any logged in user can change the common word flag ✅
There is a login page/component/popup/etc for users to sign in ✅
Only users over 21 years of age (based on birthday) with a claim of MasterOfTheUniverse can add and remove words ❌ - I could seemingly add words when not logged in
Users can be set up at start up in the API project as a seed (apriori), there does not need to be a sign up mechanism ✅

**Comment by VForev** [comment]
If the link to the website is bugging out, refresh the page a few times, I don't know why but eventually it works. But if you can't get it at all then you can use this one, this is a link to my repo site which is the same thing just running on my Azure. 
[https://white-flower-0e3bf9b1e.3.azurestaticapps.net/](url)

File: Wordle.Api/Wordle.Api/Controllers/WordController.cs [hunk 1]
```diff
@@ -35,6 +36,13 @@ public async Task<Word> AddWord(string newWord, bool isCommon)
             return await _wordService.AddWordAsync(newWord, isCommon);
         }
 
+        [HttpDelete("Delete")]
```
Comment (meghanmae) [inline Wordle.Api/Wordle.Api/Controllers/WordController.cs:39]: You need auth policies on these

**Pull request closed without merging**